e2e/setup/global-setup.ts

```
@@ -0,0 +1,238 @@
+  const entries = Object.values(USERS)
+  await Promise.all(
+    entries.flatMap(host =>
+      entries
+        .filter(peer => peer.instance !== host.instance)
+        .map(peer => createContact(host, peer))
+    )
+  )
+}
+
+export default async function globalSetup(): Promise<void> {
+  console.log('[e2e] Cleaning up previous containers...')
```

ESLint

[error] Unexpected console statement.

(no-console)